src/Utils/InitialConditions/InitialConditions.jl

```diff
@@ -0,0 +1,15 @@
+module InitialConditions
+
+using UnPack
+
+using CLIMAParameters
+using ClimaCore: Geometry, Spaces, Fields
+
+include("init_1d_ekman_column.jl")
+include("init_2d_density_current.jl")
+include("init_2d_rising_bubble.jl")
+include("init_3d_rising_bubble.jl")
+include("init_3d_solid_body_rotation.jl")
+include("init_3d_baroclinic_wave.jl")
```

Comment:
how about 3dbox and 3dsphere in the file name to differentiate. I guess no need for 1d and 2d because we only have the box